pages/en/deploying/subgraph-studio-faqs.mdx

```diff
@@ -8,7 +8,7 @@ title: Subgraph Studio FAQs
 
 ### 2. How do I create an API Key?
 
-In the Subgraph Studio, you can create API Keys as needed and add security settings to each of them.
+To create an API, navigate to the Subgraph Studio and connect your wallet. You will be able to click the API keys tab at the top. There, you will be able to create an API key by clicking the button.
```

Comment:
"Clicking the button" sounds a bit odd. 

I think it would be better leaving out that part (The button is clearly visible) 
"There, you will be able to create an API key"

Alternatively "There, you will be able to create an API key by clicking the button labelled "Create API Key"" But this is a mouthful, and isn't really needed, as the button is very visible and self-explanatory

Reply:
Agreed! Good call.